instrumentation/netty/netty-4.1/library/src/main/java/io/opentelemetry/instrumentation/netty/v4_1/NettyClientTelemetry.java

```diff
@@ -72,6 +72,6 @@ public ChannelInboundHandlerAdapter createResponseHandler() {
    * request executed on a {@link Channel}.
    */
   public static void setChannelContext(Channel channel, Context context) {
-    channel.attr(AttributeKeys.WRITE_CONTEXT).compareAndSet(null, context);
+    channel.attr(AttributeKeys.WRITE_CONTEXT).set(context);
```

Comment:
`WRITE_CONTEXT` can contain `Context.root()` as well, it's not just the null value